internal/cmd/kafka/command_cluster_update.go

```diff
@@ -72,16 +73,20 @@ func (c *clusterCommand) update(cmd *cobra.Command, args []string, prompt form.P
 			return errors.New(errors.NonEmptyNameErrorMsg)
```

Comment:
We should be able to get rid of this check. I gave this a try and the backend returned a helpful error message.

Reply:
what's the msg you're seeing? I tried setting name to "" and it doesn't error, it just don't update.

Reply:
```
% confluent kafka cluster update lkc-1j9gqv --name ""
Error: failed to update Kafka cluster: Either name or config must be set for update request.: 422 Unprocessable Entity

Suggestions:
    A cluster can't be updated while still provisioning. If you just created this cluster, retry in a few minutes.
```

Reply:
that's because you didn't set cku update. If you set a cku update and name to "" at the same time, it'll just update the cku and the name won't be changed.

Reply:
ah that's a backend bug then

Reply:
Ah, looks like we should keep this error message then.